proof/refine/X64/CSpace_R.thy

```diff
@@ -4876,6 +4786,15 @@ lemma maskedAsFull_revokable_safe_parent:
 
 context begin interpretation Arch . (*FIXME: arch-split*)
 
+(* FIXME arch-split: generic statement, arch specific proof *)
```

Comment:
ok to leave here, but I should ask whether there's a better place...

Reply:
I was going to leave that question for you :-). There probably is eventually, but for now I think leaving it in this theory is good.

Reply:
(Will leave this in for later when we get there in the arch split process)